### DISCUSSION TITLE
Changing PrimeNg Version 16 p-table paginator icons

### DISCUSSION THREAD

**Discussion by GiangNguyenDev** [primefaces/community#68] [PrimeNG] [answered]
Hello,

I want to change the next, previous, last, and first paginator icons in the PrimeNg version 16 p-table. In version 16, all icons are displayed as SVG. But, I want to replace SVG icons with PrimeNg icons. How can I do that? Many thanks.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've introduced icon templating for all PrimeNG components, including the paginator.
You can customize the icons by templating options below.

```
paginatorfirstpagelinkicon
paginatorlastpagelinkicon
paginatorpreviouspagelinkicon
paginatornextpagelinkicon

```
Here is a Stackblitz [example](https://stackblitz.com/edit/gdngln?file=src%2Fapp%2Fdemo%2Ftable-paginator-basic-demo.html) with icon templating. I've created this example by using PrimeIcons but you can use any other library.

For more info please visit the [paginator API documentation](https://primeng.org/paginator#api.paginator.templates)

**Comment by GiangNguyenDev** [comment]
Hi,
Thank you for your quick response. Unfortunately, I cannot open your Stackblitz link (it redirects to Stackblitz homepage). Can you check it please?

**Reply by cetincakiroglu** [reply]
Hi,

Sorry for that. Link updated.

**Reply by GiangNguyenDev** [reply]
It works now. Thank you for your help 🙂.

**Comment by joshnarwold-mck** [comment]
I'm trying to work out how to override the new SVG icons with Font Awesome using my Prime NG theme so I don't have to insert a customization in the HTML for each instance of a component. Any ideas?

(In previous versions of Prime NG, I had an icons.scss file that replaced the character for each icon with the corresponding Font Awesome character.)
